Instagram-Clone/Instagram-Clone/Global/Extension/UITabBarController+.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UITabBarController+.swift
+//  Instagram-Clone
+//
+//  Created by subinyoon on 2022/04/17.
+//
+
+import UIKit
+
+extension UITabBarController {
+    /// 릴스 탭 바 클릭 시 아이템 색상과 에셋 색상을 반전시키는 함수입니다
+    func tabBarReverseCustom() {
```

Comment:
이 함수의 재사용성을 더 높이기 위해서는 파라미터로 index를 받아와서 해당 인덱스의 탭 바의 속성을 reverse하는것이 어떨까요..?
```suggestion
    func tabBarReverseCustom(index: Int) {
    if selectedIndex == index { ... }
    
```
이른 느낌으로,,,

Reply:
감사합니다 교수님 👑 👑 👑
갓 블레스 유